.circleci/config.yml

```diff
@@ -118,7 +122,7 @@ jobs:
 
   debian-9: &DEBIAN
     docker:
-      - image: "tahoelafsci/debian:9"
+      - image: "tahoelafsci/debian:9-py2.7"
```

Comment:
I think you're close to the truth with the last description.  It's only our CircleCI jobs that will update these images.  There's a nightly job that does it but only the CircleCI configuration in master is respected for those jobs - so the code in master was used to update `tahoelafsci/debian:9` sometime within the last 24 hours (I forget exactly when they run) and the nightly job to build images using the code in this PR is *never* run automatically - but I have forced it to run by futzing with the CircleCI configuration during development so that it is a PR-triggered job instead of a nightly-triggered job.

`${DISTRO}:${TAG}-py${PYTHON_VERSION}` is indeed the new tag scheme.  The old tags will not be used once this PR is merged and the CircleCI nightly image buildings jobs will begin tagging their results using this scheme instead.

The old tags will linger until someone explicitly deletes them, I believe - which I probably wouldn't do too soon because (a) there might be other active PRs that still use them and (b) the storage is free.


This is all pretty